### PR TITLE
cut

### PR DESCRIPTION
**Pull request by Quangdung090**
dasdas

### CODE REVIEW

**Comment by Duc3m** [comment]
m code van nhu con cac

File: file1.txt
Comment (Duc3m) [inline file1.txt:1]: xoa cai dit me may a

File: file2.txt
Comment (Duc3m) [inline file2.txt:1]: code ngu vl

**Review by Quangdung090 — COMMENTED** [review]
deasd

File: file1.txt
Comment (Quangdung090) [inline file1.txt:1]: dasdasd

File: file2.txt
Comment (Quangdung090) [inline file2.txt:1]: dasdasd

**Review by Quangdung090 — COMMENTED** [review]
.

**Review by Duc3m — CHANGES_REQUESTED** [review]
sua cho nay ho bo may cai

**Review by Duc3m — COMMENTED** [review]
code nhu lon

**Comment by Quangdung090** [comment]
> code nhu lon

**Review by Duc3m — APPROVED** [review]

**Review by Duc3m — DISMISSED** [review]
sua tiep

**Review by Duc3m — APPROVED** [review]
code khon r do

**Review by Duc3m — COMMENTED** [review]
lai code ngu

**Review by Duc3m — APPROVED** [review]

**Pull request merged**